### PR TITLE
Scenes update

### PR DESCRIPTION
**Pull request by pleszkiewicz**
Update all the rest scenes/components/props.

### CODE REVIEW

File: src/test_scenes/sceneScrollView.js [hunk 1]
```diff
@@ -1,37 +1,24 @@
...
+    this.state = { isVertical: true, scrollValue: 0 };
   }
 
+  onOrientationChanged = event => this.setState({ isVertical: event.On, scrollValue: 0 });
   onScrollChanged = event => {
     console.log('onScrollChanged: ', event.ScrollValue);
     this.setState({ scrollValue: event.ScrollValue });
   }
 
-  onTopLeftButtonClick = () => { this.setState({ alignment: 'top-left' }); }
```
Comment (kpiascik) [inline src/test_scenes/sceneScrollView.js:20]: why were these removed?

File: src/test_scenes/sceneScrollBar.js [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
           defaultItemPadding={[0.07, 0, 0.07, 0]}
-          orientation={'horizontal'}>
+          orientation={Orientation.horizontal}>
           <ScrollBar thumbSize={0.25} thumbPosition={0.33} length={2.0} thickness={0.04} rotation={quat} />
           <LinearLayout>
-            <LinearLayout defaultItemPadding={[0.07, 0.07, 0.07, 0.07]} orientation={'horizontal'}>
+            <LinearLayout defaultItemPadding={[0.07, 0.07, 0.07, 0.07]} orientation={Orientation.horizontal}>
               <ScrollBar thumbPosition={0.1} length={1.2} thickness={0.07} rotation={quat} />
-              <ScrollBar thumbSize={7.0} thumbPosition={0.33} length={0.7} thickness={0.11} rotation={quat} />
+              <ScrollBar thumbSize={1.0} thumbPosition={0.33} length={0.7} thickness={0.11} rotation={quat} />
```
Comment (kpiascik) [inline src/test_scenes/sceneScrollBar.js:18]: why was this 7 before?

File: src/test_scenes/sceneScrollBar.js [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
             </LinearLayout>
             <LinearLayout defaultItemPadding={[0.07, 0.07, 0.07, 0.07]}>
-              <ScrollBar thumbPosition={0.1} length={1.2} thickness={0.07} orientation={'horizontal'} />
-              <ScrollBar thumbSize={0.33} thumbPosition={0.33} length={1} thickness={0.11} orientation={'horizontal'} />
-              <ScrollBar thumbSize={0.5} thumbPosition={2.0} length={1.2} thickness={0.08} orientation={'horizontal'} />
+              <ScrollBar thumbPosition={0.1} length={1.2} thickness={0.07} orientation={Orientation.horizontal} />
+              <ScrollBar thumbSize={0.33} thumbPosition={0.33} length={1} thickness={0.11} orientation={Orientation.horizontal} />
+              <ScrollBar thumbSize={0.5} thumbPosition={1.0} length={1.2} thickness={0.08} orientation={Orientation.horizontal} />
```
Comment (kpiascik) [inline src/test_scenes/sceneScrollBar.js:23]: changed from 2 to 1?

File: src/test_scenes/sceneScrollView.js (hunk 1 above)
Comment (pleszkiewicz) [reply src/test_scenes/sceneScrollView.js:20]: These were event handlers for setting alignment (new anchorPoint) of ScrollView. I removed this panel from the scene (we have a different scene for testing alignment).

File: src/test_scenes/sceneScrollBar.js (hunk 2 above)
Comment (pleszkiewicz) [reply src/test_scenes/sceneScrollBar.js:18]: I don't know why there was 7 before, but only values between 0 and 1 are valid. 0 means a minimum size of the thumb (a dot), 1 means a maximum value (full length of the scrollbar). I set PropType which validates if the `thumbSize` value is within a [0,1] range.

File: src/test_scenes/sceneScrollBar.js (hunk 3 above)
Comment (pleszkiewicz) [reply src/test_scenes/sceneScrollBar.js:23]: The same with `thumbPosition` - the valid values are within a range [0, 1].

**Review by kpiascik — APPROVED** [review]
